pom.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
-  ~ Copyright (c) 2019-2023, WSO2 LLC. (http://www.wso2.com).
+  ~ Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Any reason to change this?

Reply:
Fixed with https://github.com/wso2/identity-api-server/pull/473/commits/5c51c9eff45d1c55a16c7858a056b5a9eed2ce86